README.md

```diff
@@ -101,6 +101,33 @@ It's also convenient for profiling, debugging, etc, especially since all benchma
 ruby benchmarks/some_benchmark.rb
 ```
 
+### Benchmark organization
+
+Benchmarks can be organized in three ways:
+
+1. **Standalone .rb files** - Place a `.rb` file directly in the `benchmarks/` directory:
+   ```
+   benchmarks/fib.rb         # Benchmark name: "fib"
+   ```
+
+2. **Single benchmark per directory** - For benchmarks that need additional files (like Gemfiles):
+   ```
+   benchmarks/erubi/
+     benchmark.rb            # Benchmark name: "erubi"
+     Gemfile
+   ```
+
+3. **Multiple benchmarks per directory** - For related benchmarks sharing dependencies:
+   ```
+   benchmarks/addressable/
+     benchmark-equality.rb   # Benchmark name: "addressable-equality"
+     benchmark-join.rb       # Benchmark name: "addressable-join"
```

Comment:
This seems like a fair naming convention, but it's a bit harder than now to come up with the benchmark name "addressable-join" (which I need to use when I want to run it with `run_benchmarks.rb`) when I look at the file name `benchmarks/addressable/benchmark-join.rb` because "benchmark-" is in the middle of them.

Do you think it'd be problematic to use `benchmarks/addressable/join.rb` for this case? Like the `benchmarks/xxx.rb` files, I don't think we necessarily need to keep `benchmark` in the file names, and we usually don't have non-benchmark `.rb` files in a benchmark subdirectory anyway.

Reply:
> Do you think it'd be problematic to use `benchmarks/addressable/join.rb` for this case?

Not really. I will change to that. It is better.